src/containers/Transactions/test/Description.test.js

```diff
@@ -37,15 +36,6 @@ describe('Description container', () => {
     wrapper.unmount();
   });
 
-  it('renders description for OfferCreate', () => {
```

Comment:
Why was this test removed?

Reply:
It was moved to its own file OfferCreateDescription.test.tsx